pkg/charts/multicluster/values.go

```diff
@@ -9,27 +9,35 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
-const helmDefaultChartDir = "linkerd2-multicluster-remote-setup"
+const helmDefaultChartDir = "linkerd2-multicluster"
 
 // Values contains the top-level elements in the Helm charts
 type Values struct {
-	CliVersion          string `json:"cliVersion"`
-	GatewayName         string `json:"gatewayName"`
-	Namespace           string `json:"namespace"`
-	IdentityTrustDomain string `json:"identityTrustDomain"`
-	IncomingPort        uint32 `json:"incomingPort"`
-	LinkerdNamespace    string `json:"linkerdNamespace"`
-	ProbePath           string `json:"probePath"`
-	ProbePeriodSeconds  uint32 `json:"probePeriodSeconds"`
-	ProbePort           uint32 `json:"probePort"`
-	ProxyOutboundPort   uint32 `json:"proxyOutboundPort"`
-	ServiceAccountName  string `json:"serviceAccountName"`
-	NginxImageVersion   string `json:"nginxImageVersion"`
-	NginxImage          string `json:"nginxImage"`
-	LinkerdVersion      string `json:"linkerdVersion"`
-	CreatedByAnnotation string `json:"createdByAnnotation"`
-	LocalProbePath      string `json:"localProbePath"`
-	LocalProbePort      uint32 `json:"localProbePort"`
+	CliVersion                     string `json:"cliVersion"`
+	ControllerComponentLabel       string `json:"controllerComponentLabel"`
+	ControllerImage                string `json:"controllerImage"`
+	ControllerImageVersion         string `json:"controllerImageVersion"`
+	CreatedByAnnotation            string `json:"createdByAnnotation"`
+	Gateway                        bool   `json:"gateway"`
+	GatewayLocalProbePath          string `json:"gatewayLocalProbePath"`
+	GatewayLocalProbePort          uint32 `json:"gatewayLocalProbePort"`
+	GatewayName                    string `json:"gatewayName"`
+	GatewayNginxImage              string `json:"gatewaynginxImage"`
+	GatewayNginxImageVersion       string `json:"gatewaynginxImageVersion"`
```

Comment:
Wrong casing in `gatewaynginxImage` is preventing the gateway image to being properly set in `gateway.yaml`

Reply:
... `gatewaynginxImageVersion`

Reply:
I fixed this in the piggyback, will fix it here as well